techniques/failures/F96.html

```diff
@@ -2,148 +2,114 @@
          </p>
       </section>
       <main>
          <section id="applicability">
-            <h2>Applicability</h2>
-            <p>All technologies that include interactive controls (such as links or form inputs).</p>
-            <p>This technique relates to <span><a href="https://waic.jp/docs/WCAG21/Understanding/label-in-name">Success Criterion 2.5.3: Label in Name</a> (Failure)</span>.
-            </p>
+            <h2>適用（対象）</h2>
+            <p>インタラクティブなコントロール (リンクやフォーム入力など) を含むすべてのテクノロジー。</p>
+            <p>この達成方法は、<span><a href="https://waic.jp/docs/WCAG21/Understanding/label-in-name">達成基準 2.5.3: 名前 (name) のラベル (不適合) </span>に関連する。</p>
          </section>
          <section id="description">
-            <h2>Description</h2>
-            <p>The objective of this Failure is to describe situations where speech input users cannot
-               reliably speak the name of a control because it differs from the visible label.
-            </p>
-            <p>When speech input users interact with a web page, they usually speak a command followed
-               by the reference to some visible label (like the text in a button or a link, or the
-               text labelling some input). If the visible label does not match the accessible name
-               of the control, speech users may be unable to directly activate that control.
-            </p>
-            <p>There are techniques that go beyond the visible label text of elements. The idea is
-               to provide more context for users of assistive technologies to whom the visible context
-               may not be available. Examples are the use of accessible hidden (e.g., offscreen-positioned)
-               text, the use of the aria-label attribute to overwrite the label text, or the use
-               of the aria-labelledby attribute to compose a label text from one or more strings
-               strings on the page.
-            </p>
-            <p>When the use of these techniques results in an accessible name in which the exact
-               string of the visible label does not occur in the accessible name, speech users may
-               be unable to activate that control. Refer to the <a href="https://www.w3.org/TR/accname/">Accessible Name and Description Computation</a> algorithm to work out the order of precedence in computing the accessible name of
-               a control.
-            </p>
+            <h2>解説</h2>
+            <p>この失敗例の目的は、コントロールの名前と、目に見えるラベルが異なっているために、音声入力のユーザーがコントロールの名前を正しく話せないという状況を説明することである。</p>
+            <p>音声入力のユーザは、Webページを操作するとき、通常はコマンドの後に何らかの表示ラベル(ボタンやリンクの中のテキストや、入力欄にラベリングしてあるテキスト)への参照を続けて話す。（永野注: 日本語でも”コマンド→対象”の順で話すのかな？）表示されるラベルが、コントロールのアクセシブルな名前と一致しない場合、音声を使うユーザーは、そのコントロールを直接アクティブにできないことがある。</p>
```

Comment:
```suggestion
            <p>音声入力の利用者がウェブページを操作するとき、通常はコマンドの後に何らかの表示ラベル (ボタンやリンクの中のテキスト、入力欄にラベル付けしてあるテキスト) への参照を続けて話す。表示されるラベルが、コントロールのアクセシブルな名前 (accessible name) と一致しない場合、音声を使う利用者は、そのコントロールを直接アクティブにできないことがある。</p>
```

個人的なコメントは訳文に記載せず、issueのコメントに記載するなどしてください。「日本語でも”コマンド→対象”の順で話すのか」については私もよくわかっていません。

初出でない箇所も  (accessible name)  は付与します。